integrations/swing.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: TinyMCE Swing integration
+title_nav: Swing
+description: Seamlessly integrates TinyMCE into Swing applications.
+keywords: integration integrate swing
+---
+
+Users can easily configure TinyMCE in Swing through the TinyMCE Swing Integration. 
+
+### Buy TinyMCE Swing integration
```

Comment:
We've used the product name "TinyMCE for Swing" in other text.

Reply:
Fixed